### DISCUSSION TITLE
Antd Select组件可以通过JSON Schema异步联动数据的方式使用吗？

### DISCUSSION THREAD

**Discussion by isunkui** [alibaba/formily#2317] [Q&A 2.x] [answered]
文档中只有[Markup Schema](https://antd.formilyjs.org/zh-CN/components/select#markup-schema-%E5%BC%82%E6%AD%A5%E8%81%94%E5%8A%A8%E6%95%B0%E6%8D%AE%E6%BA%90%E6%A1%88%E4%BE%8B)的异步联动数据源案例，JSON Schema的使用案例没有提供。

**Comment by janryWang** [comment, marked as answer]
Markup Schema和JSON Schema完全是等价的
